scripts/gdal/gdal_preset.py

```diff
@@ -132,3 +130,10 @@ def get_alpha_command() -> List[str]:
         # Ensure the target has a alpha channel
         "-dstalpha",
     ]
+
+
+def get_transform_srs_command(target_epsg: str) -> List[str]:
+    """
+    Get a "Gdalwarp" command to transform the srs
+    """
+    return ["gdalwarp", "-t_srs", f"EPSG:{target_epsg}"]
```

Comment:
What resampling method does `gdalwarp` run by default? I believe we need to use `bilinear` (?)

Reply:
yes I was just thinking this, shall we match Rebecca's gist:
`gdalwarp -t_srs 'EPSG:2193' -r bilinear -tr 10 -10 -co compress=lzw ./output/overall_wgs84.tif ./output/overall_NZTM.tif`

Reply:
this is also missing the rest of the arguments, like `ALL_CPUS` that I shared earlier.